### PR TITLE
Allow 'bird' or 'birb' as answer for normal/flying type

### PR DESCRIPTION
**Pull request by Aegyo**
(is this the right branch?)

can't do `(${forward}|${reverse}|${meme})` when blank meme because it matches on the empty strings in between characters 😞

### CODE REVIEW

File: other/quiz/quiz_questions.js [hunk 1]
```diff
@@ -197,7 +197,11 @@ const pokemonType = () => new Promise(resolve => {
   (async () => {
     const pokemon = getRandomPokemon();
     const types = pokemon.type.map(t => PokemonType[t]);
-    const answer = new RegExp(`^\\W*(${types.join('\\W*')}|${types.reverse().join('\\W*')})\\b`, 'i');
+    let memeAnswer = null;
+    if (types.includes('Flying') && types.includes('Normal')) {
+      memeAnswer = 'bir[bd]';
+    }
+    const answer = new RegExp(`^\\W*(${types.join('\\W*')}|${types.reverse().join('\\W*')}${memeAnswer && `|${memeAnswer}`})\\b`, 'i');
```
Comment (RedSparr0w) [inline other/quiz/quiz_questions.js:204]: I think this outputs `null` as an answer when `memeAnswer` is null

**Comment by RedSparr0w** [comment]
Yeah it's the correct branch 👍

File: other/quiz/quiz_questions.js [hunk 2]
```diff
@@ -197,7 +197,11 @@ const pokemonType = () => new Promise(resolve => {
   (async () => {
     const pokemon = getRandomPokemon();
     const types = pokemon.type.map(t => PokemonType[t]);
-    const answer = new RegExp(`^\\W*(${types.join('\\W*')}|${types.reverse().join('\\W*')})\\b`, 'i');
+    let memeAnswer = null;
```
Comment (RedSparr0w) [inline other/quiz/quiz_questions.js]: I think setting this to a blank string resolves it though?
```suggestion
    let memeAnswer = '';
```

File: other/quiz/quiz_questions.js (hunk 1 above)
Comment (Aegyo) [reply other/quiz/quiz_questions.js:204]: oh damn you are right my bad

**Comment by Aegyo** [comment]
Any idea what is happening with the eslint fail?

File: other/quiz/quiz_questions.js (hunk 1 above)
Comment (Aegyo) [reply other/quiz/quiz_questions.js:204]: commited your suggestion

**Review by RedSparr0w — APPROVED** [review]
Looks good, thanks for the PR! 👍

**Comment by RedSparr0w** [comment]
Not too sure tbh, something to do with a config file though 😅

**Pull request merged**